Scripts/fetch-prebuilt-wda.js

```diff
@@ -16,14 +14,15 @@ async function fetchPrebuiltWebDriverAgentAssets () {
   log.info(`Getting WDA release ${downloadUrl}`);
   let releases;
   try {
-    releases = await request.get(downloadUrl, {
+    releases = (await axios({
+      url: downloadUrl,
       headers: {
```

Comment:
btw, is it better to define default headers in appium-support?
Mostly it should be `user-agent` and `accept: */*', but we may forget to add them in a new place

Reply:
the helper in appium-support only downloads the content, which means the Accept header does not really matter there, unlike here, where we explicitly want to get json response from the server. Regarding user-agent: I'd rather leave it to the client to decide whether to include it or not. Perhaps, we could add an option there whether to add it by default.